ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -76,6 +76,53 @@ tests:
         NUM_MASTERS=1
         OPENSHIFT_INSTALL_RELEASE_IMAGE=registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.2.0
     workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-static-ip-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        ADDITIONAL_PARAMS="--with-static-ips"
+    workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-single-node-static-ip-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        NUM_MASTERS=1
+        ADDITIONAL_PARAMS="--with-static-ips"
+        OPENSHIFT_INSTALL_RELEASE_IMAGE=registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.2.0
+    workflow: baremetalds-assisted
+- artifact_dir: /tmp/artifacts
+  as: e2e-metal-assisted-single-node-ipv6-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    env:
+      ASSISTED_CONFIG: |
+        NUM_MASTERS=1
+        IPv4=no
+        IPv6=yes
+        PROXY=yes
```

Comment:
Didn't we say we don't want proxy?

Reply:
I thought about changing the var's behavior to use reverse-proxy, but maybe you're right and it's a bit confusing
@empovit WDYT?